kliff/transforms/configuration_transforms/graphs/kim_driver_graph_data.cpp

```diff
@@ -0,0 +1,297 @@
+/*
+ * Contains functions to generate complete graph from a given set of
+ * coordinates and species. The generated graph uses the neighbor list
+ * code from the kliff.nl module, but instead of linking against it, it
+ * is copied here. This is done to avoid the need complicated build steps
+ * and to make the graph module self sufficient.
+
+ * Author: Amit Gupta
+ * Email: gupta839@umn.edu
+
+ * License: LGPL v2.1, This is distributed under the original KLIFF license.
+ */
+
+#include <cstddef>
+#include <cstdint>
+#include <iostream>
+#include <string>
+#include <vector>
+#include <set>
+
+#include <pybind11/pybind11.h>
+#include <pybind11/numpy.h>
+#include <pybind11/stl.h>
+
+#include "neighbor_list.h"
```

Comment:
Although you've added something in `setup.py`, but here we are still directly accessing it, without providing any path. Not sure whether this works or not.

Reply:
In the latest setup.py it works.